src/_DataStructures_/Trees/Trie/total-words-in-trie/total-words-in-trie.test.js

```diff
@@ -0,0 +1,35 @@
+const totalWordsInTrie = require('./index')
+const Trie = require('../index');
+var assert = require('assert');
```

Comment:
Why var?

Reply:
Just like that, do I change it to const?